Keyboards/KeyboardsBase/ScribeFunctionality/Conjugate.swift

```diff
@@ -252,6 +252,12 @@ func returnConjugation(keyPressed: UIButton, requestedForm: String) {
       proxy.insertText(wordToReturn + " ")
     }
   }
+  if controllerLanguage == "German" {
+    let components = wordToReturn.components(separatedBy: " ")
+    if components.count == 2 {
+      proxy.adjustTextPosition(byCharacterOffset: (components[1].count + 1) * -1)
+    }
+  }
```

Comment:
Also, perhaps a question more for you, @andrewtavis

Was wondering as well - could the functions throughout the code be modified to be more generic/language-agnostic? I'm thinking generally that any functionality that's pretty universal can stay where they are, but for instance, German-specific checks/logic can go under `Keyboards/LanguageKeyboards/German/*.swift`. Thought of this as I was thinking code may get messy/hard-to-read with language-specific logic sprinkled around, especially as more languages are added over time (and their exceptions and quirks along with them).

**Note:** This doesn't have to be a part of this PR; can be done later as it might require some refactoring. Just curious on thoughts on the above.

Reply:
Met offline with @andrewtavis and @SaurabhJamadagni on the above. There's consensus that refactoring language-specific logic out from the base keyboard should be something Scribe should look to. Created the issue #323 to cover for this. Please anyone feel free to add any thoughts or ideas there! :smile: 

Also, another good idea could be to keep this refactoring in mind as other work is done throughout the project. If feasible to implement language-specific logic already separated out, do so :rocket: